brainglobe_segmentation/regions/analysis.py

```diff
@@ -33,12 +34,12 @@ def region_analysis(
             )
     if summarise:
         if output_csv_file is not None:
-            print("Summarising regions")
+            show_info("Summarising regions")
             summarise_brain_regions(
                 label_layers, output_csv_file, atlas.resolution
             )
 
-    print("Finished!\n")
+    show_info("Finished!\n")
```

Comment:
The new line part (`\n`) is no longer needed. This was just to add a new line within the console.